### DISCUSSION TITLE
`@vercel/node` VercelResponse type doesn't support header property

### DISCUSSION THREAD

**Discussion by lorypelli** [vercel/community#3244] [Help]
### Summary

What can I do about this? Should I use `Response` instead of `VercelResponse`?

### Steps to Reproduce

I'm just using `@vercel/node` for my serverless discord bots

**Comment by swarnava** [comment]
Hi,

What are you trying to accomplish?

**Reply by lorypelli** [reply]
I had an idea to fix, but I'm not able to test it right now, I will let you know
